python/cudf/cudf/pandas/scripts/conftest-patch.py

```diff
@@ -8040,6 +8040,7 @@ def pytest_unconfigure(config):
     "tests/window/moments/test_moments_consistency_rolling.py::test_rolling_apply_consistency_sum[all_data6-rolling_consistency_cases1-True-sum]",
     "tests/window/moments/test_moments_consistency_rolling.py::test_rolling_apply_consistency_sum[all_data7-rolling_consistency_cases1-False-sum]",
     "tests/window/moments/test_moments_consistency_rolling.py::test_rolling_apply_consistency_sum[all_data7-rolling_consistency_cases1-True-sum]",
+    "tests/window/moments/test_moments_consistency_rolling.py::test_rolling_consistency_series_cov_corr[series_data1-rolling_consistency_cases0-False-0]",
```

Comment:
Is this failure flaky, or does it fail every time? If it's flaky you'll need to put this in one of the other lists because if you put it in this list pytest will mark this as XPASS when it passes, which in turn will trigger an overall test suite failure because we run with xfail_strict.